proto/cosmos/tx/v1beta1/tx.proto

```diff
@@ -228,7 +228,7 @@ message Tip {
 // actual tx). AuxSignerData is not a valid tx in itself, and will be rejected
 // by the node if sent directly as-is.
 //
-// Since: cosmos-sdk 0.45
+// Since: cosmos-sdk 0.46
 message AuxSignerData {
```

Comment:
I just thought of 3 more places:
- in `SignMode`, for `SIGN_MODE_DIRECT_AUX`
- `message Tip`
- and `message SignDocDirectAux`

Reply:
added